docker-compose.yml

```diff
@@ -0,0 +1,88 @@
+version: "3.7"
+
+services:
+  server:
+    image: shieldbattery/${SERVER_NAME}:${SERVER_VERSION:-latest}
+    container_name: sb-${SERVER_NAME}-server
```

Comment:
sb- prefix is probably redundant because starting these containers will use a Docker-Compose project name, even if you dont set one. See: https://docs.docker.com/compose/reference/overview/#use--p-to-specify-a-project-name

Reply:
I removed most of the `container_name` usage in this file, as it seemed fairly redundant with the service names anyway. I only left it for `nginx_proxy` because other containers need it to have a consistent ID.

Unsure if that is a typical usage of `container_name` or not, but it seems reasonable to me atm